core/domain/config_domain.py

```diff
@@ -220,31 +230,31 @@ def __init__(self, name, schema, description, default_value):
         Registry.init_config_property(self.name, self)
 
     @property
-    def name(self):
+    def name(self) -> str:
         """Returns the name of the configuration property."""
 
         return self._name
 
     @property
-    def schema(self):
+    def schema(self) -> Dict[str, Any]:
         """Returns the schema of the configuration property."""
 
         return self._schema
 
     @property
-    def description(self):
+    def description(self) -> str:
         """Returns the description of the configuration property."""
 
         return self._description
 
     @property
-    def default_value(self):
+    def default_value(self) -> Any:
         """Returns the default value of the configuration property."""
 
         return self._default_value
 
     @property
-    def value(self):
+    def value(self) -> Any:
```

Comment:
Explain why this returns `Any`

Reply:
If any of the condition is not satisfied then we are returning `default_value` which is of `Any` type

Reply:
ditto as above

Reply:
done!